backend/app/models/open_call.rb

```diff
@@ -0,0 +1,22 @@
+class OpenCall < ApplicationRecord
+  extend FriendlyId
+  friendly_id :investor_prefixed_name, use: :slugged
+
+  belongs_to :investor
+
+  validates :instrument_type, inclusion: {in: InstrumentType::TYPES, allow_blank: true}, presence: true
```

Comment:
Yeah, to keep integrity with `InstrumentType::TYPES` is gonna be slight headache -- no big deal, but if you would have table for `InstrumentType::TYPES`, you would get this for free. Sry, had to mention it :D

Reply:
Everything has pros and cons.